great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -53,8 +70,8 @@ class GeCloudStoreBackend(StoreBackend, metaclass=ABCMeta):
         },
     }
 
-    RESOURCE_PLURALITY_LOOKUP_DICT: bidict = bidict(
-        **{
+    RESOURCE_PLURALITY_LOOKUP_DICT: bidict = bidict(  # type: ignore[misc]
```

Comment:
when we use the `# type: ignore` comment, should we add some context around why we're skipping the check? I'm good either way but we should probably establish a standard for the team.

Reply:
Yes, this particular error for `bidict` is `Keywords must be string`.
I think we actually need 2 different standards. One as we are trying to get full type coverage & another once we have full coverage.

Once we have full type coverage I think this should be the standard.
But we are sitting on over 2k typing errors now, so I think requiring this extra step now will mean it never gets done.